### ISSUE TITLE
How to guarantee the relative size between different quantiles in MQ-RNN?

### ISSUE THREAD

**Issue by Goofy321** [jingw2/demand_forecast#14]
There is a fixed relationship between different quantiles. For example, P90 should be bigger than P10. So how to guarantee this relationship in MQ-RNN, where different quantiles are forecasted at the same time?

**Comment by jingw2** [comment]
What I understand is to use quantile loss to control the relative relationship between quantiles. MQ-RNN could directly output results of different quantiles from the network.

**Comment by Goofy321** [comment]
Yes. I can understand that  by using quantile loss as the objective function, the network is trained in the direction of predicting different quantiles. And we believe a well-trained network can meet the requriment of the relative relationship between quantiles most of the time. What worries me is whether the network can output quantiles with expected relative relationship always? I mean, for every sample.

**Comment by 2017wxyzwxyz** [comment]
@Goofy321，Please send me your code, thank you very much！ My e-mail address is NewWxy2016@163.com
